scripts/make_release_branch.py

```diff
@@ -1,15 +1,62 @@
-from common import get_patch_version, inc_patch_version, GRADLE_PROPERTIES
+import argparse
+import datetime
+import json
+import re
+from urllib.request import urlopen
+
+import github_connect
+from common import get_patch_version, inc_patch_version, GRADLE_PROPERTIES, env
 from git import git_command
+from make_next_milestone import RELEASE_MANAGER_RE
+
+
+def send_slack_message():
+    slack_ids = json.loads(args.slack_ids)
+
+    milestone = github_connect.get_current_milestone(repo=env("GITHUB_REPOSITORY"))
+
+    release_version = milestone["title"]
+    release_manager = re.search(RELEASE_MANAGER_RE, milestone['description']).group(1)
+    link = milestone['html_url']
+    date = datetime.datetime.strptime(milestone["due_on"], "%Y-%m-%dT%H:%M:%SZ").date()
+
+    slack_user = slack_ids.get(release_manager)
+    message = {
+        "release_version": release_version,
+        "link": link,
+        "date": f'%s.%s.%s' % (date.day, date.month, date.year),
+        "slack_user": slack_user
+    }
+
+    urlopen(args.webhook, data=json.dumps(message).encode())
+
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    # Webhook is unique URL for the Slack workflow. After sending a request to it, workflow will start doing its work.
+    # Looks like https://hooks.slack.com/workflows/{unique_part}, can be found on the workflow builder page.
+    parser.add_argument("--webhook", type=str, required=True)
+    # slack_ids is a json looks like {"github-username1": "slack-ID1",  ... }.
+    # NOTE: slack ID for user can be found on their profile > ⋮ > Copy member ID.
+    # It is a set of random digits and letters, not @name.surname or any custom usernames.
+    # On slack workflow side it will be automatically treated as @username.
+    parser.add_argument("--slack_ids", type=str, required=True)
+    args = parser.parse_args()
+
     patch_version = get_patch_version()
 
     release_branch = f"release-{patch_version}"
     git_command("branch", release_branch)
     git_command("push", "origin", release_branch)
 
+    try:
+        send_slack_message()
```

Comment:
Let's send message **after** all release actions.
Not to depend on `patch_version` on file system that is used inside `github_connect.get_current_milestone`, just pass current `patch_version` as parameter to `get_current_milestone`